app/design/frontend/Magento/luma/web/css/source/_extends.less

```diff
@@ -7,6 +7,9 @@
 //  List default styles reset
 //  ---------------------------------------------
 
+/**
+ * @codingStandardsIgnoreStart
+ */
```

Comment:
Is there a particular rule in the coding standard that needs to be adjusted, or something here that should be corrected to conform?
```suggestion
```

Reply:
yes it's shoud be remove for correct syntaxt and check validate